test-prow-e2e.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
```

Comment:
We'll need to install the plugin somewhere. Presumably in this script?

Reply:
> We'll need to install the plugin somewhere. Presumably in this script?

@spadgett @jhadvig  I will need your help on how to install the plugin in this script. Any ideas?

Reply:
You should need to run these steps to install the helm chart:

https://github.com/openshift/console-plugin-template#installing-the-helm-chart

We'll want to use the image built by CI. @jcaianirh worked on this for the demo plugin tests in console and might be able to help. Not sure if `helm` is available in the image we're using. If not, we'll need to figure out how to install it.